boards/tenstorrent/tt_grendel_smc/tt_grendel_smc_defconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2025 Tenstorrent AI ULC
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=n
+CONFIG_TENSTORRENT_VIRT_CONSOLE=y
+CONFIG_XIP=n
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=50
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=500
```

Comment:
This is probably a dumb question but why is this so much lower than the default of `1000000`